src/main/java/org/folio/rest/impl/LoginAPI.java

```diff
@@ -325,9 +311,8 @@ private Future<String> handleLogin(LoginCredentials credentials, String tenantId
             logger.debug(MULTIPLE_MATCHING_USERS_LOG, credentials.getUsername(), credentials.getUserId(),
```

Comment:
Lets change log level to warn here, basically this log possible when mod-consortia enabled and in case when we have multiple usernames in different tenants, but starting with Q release we have feature to implement uniqueness of usernames across tenants, so this log will be not appearing commonly, but in any case better to have it with warn level

Reply:
good idea, changed